app/models/node.ts

```diff
@@ -58,6 +58,8 @@ export default class Node extends BaseFileItem.extend(Validations) {
 
     @attr('fixstring') templateFrom!: string;
 
+    @attr('string') analyticsKey?: string;
```

Comment:
Is this something that we will need to add to the API? I don't see it in current API responses.

Reply:
Yeah, this should be going out with squirtle, I think. He attached it to one of Henrique's PRs.